jerry-core/CMakeLists.txt

```diff
@@ -162,6 +170,11 @@ if(FEATURE_MEM_STATS)
   set(DEFINES_JERRY ${DEFINES_JERRY} JMEM_STATS)
 endif()
 
+# Enable debugger
+if(FEATURE_DEBUGGER)
+  set(DEFINES_JERRY ${DEFINES_JERRY} JERRY_DEBUGGER)
```

Comment:
I added a check for it, if we build with python and use jerry-libc, the builder throw an error.